File: client-win-print/Client/Model/Venta.cs
using System;

namespace Client.Model {
    public class Venta {
        public int Id { get; set; }
        public int Correlat { get; set; }
        public DateTime Fecha { get; set; }
        public TipoAte TipoAte { get; set; }
        public Cliente Cliente { get; set; }
        public Totem Totem { get; set; }
    }
}

File: client-web/src/app/interfaces/venta.ts
import { Totem } from './totem';
import { Cliente } from './cliente';
import { TipoAte } from './tipo-ate';

export interface Venta {
  id: number;
  correlat: number;
  fecha: Date;
  tipoAte: TipoAte;
  cliente: Cliente;
  totem: Totem;
}
